design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -0,0 +1,361 @@
+type Query {
+
+    getNegotiableQuote(
+        id: ID!
+    ): NegotiableQuote @doc(description:"Get negotibale quote")
+
+    getAllNegotiableQuotes: [NegotiableQuote]! @doc(description: "Get all negotiable quotes")
```

Comment:
```suggestion
    getAllNegotiableQuotes: [NegotiableQuote] @doc(description: "Get all negotiable quotes")
```

Top-level, non-scalar queries should ideally be nullable.

In the example below, imagine `getAllNegotiableQuotes` is implemented by an external service. If that service goes down, and `getAllNegotiableQuotes` isn't nullable, GraphQL will fail the whole query, and discard the results of `categories`

```graphql
query MultipleOperations {
   getAllNegotiableQuotes {
      # some field selection
   }

   categories {
      items {
         name
      }
   }
}
```

Reply:
Understood, changed as suggested. Thanks for your explanation.